src/app/auth/account/additional-components/account-flows-tab/account-flows-tab.component.ts

```diff
@@ -0,0 +1,156 @@
+import { BaseComponent } from "src/app/common/base.component";
+import { ChangeDetectionStrategy, Component, Input, OnInit } from "@angular/core";
+import { Observable, combineLatest, switchMap, timer } from "rxjs";
+import { MaybeNull, MaybeUndefined } from "src/app/common/app.types";
+import {
+    CancelFlowArgs,
+    FilterByInitiatorEnum,
+    FlowsTableData,
+} from "src/app/dataset-view/additional-components/flows-component/components/flows-table/flows-table.types";
+import { DatasetFlowsService } from "src/app/dataset-view/additional-components/flows-component/services/dataset-flows.service";
+import { NavigationService } from "src/app/services/navigation.service";
+import { mockFlowSummaryDataFragments } from "src/app/api/mock/dataset-flow.mock";
+import { Dataset, FlowStatus, FlowSummaryDataFragment, InitiatorFilterInput } from "src/app/api/kamu.graphql.interface";
+import { AccountService } from "src/app/services/account.service";
+import { AccountTabs } from "../../account.constants";
+import ProjectLinks from "src/app/project-links";
+import { requireValue } from "src/app/common/app.helpers";
+
+@Component({
+    selector: "app-account-flows-tab",
+    templateUrl: "./account-flows-tab.component.html",
+    styleUrls: ["./account-flows-tab.component.scss"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AccountFlowsTabComponent extends BaseComponent implements OnInit {
```

Comment:
A lot of duplication with a similar component for datasets.
I'm sure you can do better - attempt to create a base component class that would contain identical functionality. |
Place unique dataset or account parts into subclasses.

Reply:
Added base class